lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -23,6 +23,8 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import com.fasterxml.jackson.core.type.TypeReference;
```

Comment:
Might want to prevent dynamic index creation as well (not sure about this since it's cluster wide):
```
PUT _cluster/settings
{
 "persistent": {
   "action.auto_create_index" : false
 }
}
```

Reply:
I'll have to look into this and make a separate PR for it. I have to think about how to do this as a one time operation for the entire index.

Reply:
Rethinking this - there may be other applications using the same ES install that need dynamic index creation. Maybe just document it in setup that it's preferred but not required.

Reply:
Added a note in ops.rst documentation in a separate PR